### DISCUSSION TITLE
Runway + statamic address field

### DISCUSSION THREAD

**Discussion by freshface** [statamic-rad-pack/runway#340] [Help] [answered]
Hi

I added the address field (https://statamic.com/addons/rias/address-field) to a blueprint to a runway resource.
I can add/edit an entry. It stores it as json in my database.

But on the overview table I get the following error:

```Rias\StatamicAddressField\AddressData::from(): Argument #1 ($data) must be of type ?array, string given, called in /Users/frederikheyninck/Figure8/mamp/sterpunt inclusief ondernemen/sio/vendor/rias/statamic-address-field/src/Fieldtypes/Address.php on line 25```

Is it possible to use it in combination with runway?

**Comment by freshface** [comment, marked as answer]
Ok, I added this and it works :)

```
    protected $casts = [
        'address' => 'json', // or 'json', AsArrayObject::class
    ];

```

**Reply by duncanmcclean** [reply]
Yes, you'll need to add $casts for any JSON columns.
